project.clj

```diff
@@ -96,19 +95,7 @@
                        ["@ensdomains/ethregistrar" "2.0.4"]
                        ["@ensdomains/resolver" "0.2.4"]
                        ["@sentry/node" "4.2.1"]
-                       ;; https://github.com/district0x/district-server-db/blob/3839edd/project.clj#L12
-                       ;; After deployment changes, (transitive) dependencies are not installed automatically
-                       ;; TODO: remove, they should be installed automatically
-                       [better-sqlite3 "5.4.0"]
                        [chalk "2.3.0"]
-                       ;; https://github.com/district0x/district-server-smart-contracts/blob/682e649/project.clj#L17
```

Comment:
These don't get installed for me on clean install, so I seem to need the dependencies

Reply:
:/ Ok, maybe I had package.lock or something preserved, but I thought I was doing a clean install as well...